st2common/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+# https://docs.python.org/2/distutils/sourcedist.html#commands
+# Include all files under the source tree by default.
+# Other behaviour can be used though.
+recursive-include st2common *.* *
```

Comment:
Yep, so far I suggest like this. Later on more comprehensive filtering can
be done.
BTW none code artifacts are included as you might think. Read this as
st2common/{*,*.*} in bash globing way.
MANIFEST.in is relative to the component root, i.e we include files not
under component root but only under its source folder.

On Thu, Jun 25, 2015, 08:37 Tomaz Muraus <notifications@github.com> wrote:

> In st2common/MANIFEST.in
> <https://github.com/StackStorm/st2/pull/1647#discussion_r33223754>:
>
> > @@ -0,0 +1,4 @@
> > +# https://docs.python.org/2/distutils/sourcedist.html#commands
> > +# Include all files under the source tree by default.
> > +# Other behaviour can be used though.
> > +recursive-include st2common *.* *
>
> Instead of just including all in the manifest, I would prefer to
> explicitly declare all the non-code artifacts we want to include here - so
> things like README, LICENSE, NOTICE, requirements.txt, etc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StackStorm/st2/pull/1647/files#r33223754>.
>